Content.Shared/Wieldable/WieldableSystem.cs

```diff
@@ -198,7 +197,7 @@ public bool TryWield(EntityUid used, WieldableComponent component, EntityUid use
 
         for (var i = 0; i < component.FreeHandsRequired; i++)
         {
-            _virtualItemSystem.TrySpawnVirtualItemInHand(used, user);
+            _virtualItemSystem.TrySpawnVirtualItemInHand(used, user, true);
```

Comment:
This should check if any of the item spawning failed and delete the virtual items if it does.

Reply:
Fixed in [`171e1d4` (#27975)](https://github.com/space-wizards/space-station-14/pull/27975/commits/171e1d4883ea7b98a09309e61d826e2cfaff1215)